### PR TITLE
Release/5.2.0

### PR DESCRIPTION
**Pull request by skhozin**
Updated README.md with the latest SDK version
Updated TROUBLESHOOTING GUIDE
Updated podspec with the latest SDK version.
Uploaded the latest SDK package built with Xcode 11.4.1
Uploaded a folder with images

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -276,6 +275,23 @@ Objective-C:
 - At any point you can opt a user back in by passing `false` into the same method.
 - For instructions on how to completely remove a user's data from Thunderhead ONE or Salesforce Interaction Studio - see our [api documentation](https://thunderheadone.github.io/one-api/#operation/delete).
 
+### Disable In-list Optimizations
+
+To disable in-list Optimizations, add the following to your app’s Info.plist file and setting `DisableInListOptimization` to `YES` (boolean value).
```
Comment (andreipop) [inline README.md:280]: and set

File: TROUBLESHOOTING-GUIDE.md [hunk 2]
```diff
@@ -34,3 +34,41 @@ These are **Apple error logs** logging *all* failed outgoing network connections
 When integrating the Thunderhead SDK manually into your app, you may encounter this compile error.
 
 In **Build Settings**, ensure the **Framework Search Paths** contains the framework filepath. If the framework is placed in your project directory, simply set the framework search path to $(SRCROOT) and set it to recursive.
+
+## How to resolve `UITableView` layout display issues
+
+If you come across issues with how the layout is rendered in your `UITableView`'s, try the following
```
Comment (andreipop) [inline TROUBLESHOOTING-GUIDE.md:40]: try the following:

File: TROUBLESHOOTING-GUIDE.md [hunk 3]
```diff
@@ -34,3 +34,41 @@ These are **Apple error logs** logging *all* failed outgoing network connections
...
+
+### Resolve `UITableView` layout issue by implementing `tableView:heightForRowAtIndexPath:`
+- Implement `tableView:heightForRowAtIndexPath:` delegate method in the view controller. The method implementation is required to enable the SDK swizzling implementation to call the original implementation.
```
Comment (andreipop) [inline TROUBLESHOOTING-GUIDE.md:43]: Implement the

Comment (andreipop) [inline TROUBLESHOOTING-GUIDE.md:43]: in all your view controllers where you are looking to display in-list Optimizations.

File: README.md (hunk 1 above)
Comment (skhozin) [reply README.md]: Updated, sorry missed it last time

File: TROUBLESHOOTING-GUIDE.md (hunk 3 above)
Comment (skhozin) [reply TROUBLESHOOTING-GUIDE.md]: Updated

File: TROUBLESHOOTING-GUIDE.md (hunk 2 above)
Comment (skhozin) [reply TROUBLESHOOTING-GUIDE.md]: Updated

File: TROUBLESHOOTING-GUIDE.md (hunk 3 above)
Comment (skhozin) [reply TROUBLESHOOTING-GUIDE.md]: Added

File: README.md [hunk 4]
```diff
@@ -18,6 +18,7 @@
     + [`ViewController`/`View` lifecycle overriding rules](#viewcontrollerview-lifecycle-overriding-rules)
 - [Additional features](#additional-features)
   * [Opt an end-user out of tracking](#opt-an-end-user-out-of-tracking)
+  * [Disable In-list Optimizations](#disable-in-list-optimizations)
```
Comment (andreipop) [inline README.md:21]: Disable in-list Optimizations

**Pull request merged**